### DISCUSSION TITLE
woa设备中，播放4k视频会卡加载

### DISCUSSION THREAD

**Discussion by pgs666** [ywmoyue/biliuwp-lite#820] [bug]
### Bug 描述

我的设备为8cxgen3的Huaweimatebookego，系统为Windows11 24H2，在播放4k视频的时候开始能够正常播放，在播放一段时间之后就会卡在加载界面
![image](https://github.com/user-attachments/assets/4a7817f7-3abe-4b96-b5bb-67f54555754b)
我的网络确认是正常的，因为如果同时在浏览器或者别的设备打开这个视频则不会遇到这个问题

### 操作步骤

```plain text
随意点开一个有4k画质的视频并且切换为4k
```

### 期望结果

正常播放

### 截图

![image](https://github.com/user-attachments/assets/939efd4f-47bf-48d1-988a-d535316e38f4)

### 日志

[20240918.log](https://github.com/user-attachments/files/17047545/20240918.log)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

24H2

### 设备类型

平板

### 设备架构

Arm64

### 应用架构

Arm64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
已知的播放器问题，这个暂时无法解决

**Reply by ywmoyue** [reply]
https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.4 试试4.7.4设置播放器类型为FFmpegInterop @pgs666
